mev_inspect/aave_liquidations.py

```diff
@@ -99,3 +100,23 @@ def _get_payback_token_and_amount(
                     return child_transfer.token_address, child_transfer.amount
 
     return liquidation.inputs["_collateral"], 0
+
+
+def _get_debt_data(
+    liquidation: DecodedCallTrace, child_traces: List[ClassifiedTrace], liquidator: str
+) -> Tuple[str, int]:
+    """Get transfer from liquidator to AAVE"""
+
+    for child in child_traces:
+
+        if isinstance(child, CallTrace):
```

Comment:
DecodedCallTrace is always a call trace

Reply:
This check is there for mypy purposes

Reply:
What's the failure? Because get_transfer takes a CallTrace instead of a DecodedCallTrace?
I think it also needs to be a DecodedCallTrace so better to change that type signature

Reply:
ETH transfers are call traces and often show up in the liquidations